samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Pages/htmlpage.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace TeamsMessagingExtensionsAction.Pages
+{
+    public class IndexModel : PageModel
```

Comment:
If the PageModel is not used, then the page can just be a static html page and does not require this model.

Reply:
This is not a static HTML page, it is an razor html page using the model, the sample app have 3 more capabilities.
1. loading WebView in ME action Task Module  
2. razor html page
3. razor view